lib/perl/Genome/Config/AnalysisProject/Command/AddModel.t

```diff
@@ -104,7 +104,9 @@ EOFILE
         store_only => 1,
         reprocess_existing => 0,
     );
-    my $profile_item = $add_cmd->execute();
+    my $ok = $add_cmd->execute();
+    ok($ok, 'executed add command');
+    my $profile_item = Genome::Config::Profile::Item->get(analysis_project => $analysis_project, status => 'disabled');
```

Comment:
Should we verify `$profile_item` doesn't exist before the `$add_cmd->execute()`?